pop-api/src/v0/assets/fungibles.rs

```diff
@@ -8,28 +15,22 @@ use constants::*;
 use ink::{env::chain_extension::ChainExtensionMethod, prelude::vec::Vec};
 pub use metadata::*;
 
-/// Helper method to build a dispatch call `ChainExtensionMethod` for fungibles `v0`.
-///
-/// Parameters:
-/// - 'dispatchable': The index of the module dispatchable functions.
+// Helper method to build a dispatch call `ChainExtensionMethod` for fungibles `v0`.
+//
+// Parameters:
+// - 'dispatchable': The index of the module dispatchable functions.
 fn build_dispatch(dispatchable: u8) -> ChainExtensionMethod<(), (), (), false> {
 	crate::v0::build_dispatch(FUNGIBLES, dispatchable)
 }
 
-/// Helper method to build a dispatch call `ChainExtensionMethod` for fungibles `v0`.
-///
-/// Parameters:
-/// - 'state_query': The index of the runtime state query.
+// Helper method to build a dispatch call `ChainExtensionMethod` for fungibles `v0`.
```

Comment:
```suggestion
// Helper method to build a call to read state.
```

The name of the function clearly indicates that it isnt building a dispatch call, but a call to read state. The same mistake is made in the called function (crate::v0::build_read_state and crate::v0::build_dispatch) so please also update the doc comments accordingly all the way up the stack.